gulpfile.js

```diff
@@ -1,11 +1,22 @@
-// ## Globals
```

Comment:
It makes everything look pretty when you run docco https://austinpray-web.s3.amazonaws.com/temp-roots/gulpfile.html